plugins/obs-transitions/data/luma_wipe_transition.effect

```diff
@@ -35,22 +37,31 @@ float4 PSLumaWipe(VertData v_in) : TARGET
 	float2 uv = v_in.uv;
 	float4 a_color = convert_pmalpha(a_tex.Sample(textureSampler, uv));
 	float4 b_color = convert_pmalpha(b_tex.Sample(textureSampler, uv));
-	float luma     = l_tex.Sample(textureSampler, uv).x;
+	float luma = l_tex.Sample(textureSampler, uv).x;
+	float soft = softness;
+
+	if (soft == 0.00)
+		soft += 0.01;
 
 	if (invert)
 		luma = 1.0f - luma;
 
-	float time = lerp(0.0f, 1.0f + softness, progress);
+	// user color with luma
+	float4 out_color = float4(l_color.rgb, luma);
+
+	float time = lerp(0.0f, 1.0f + soft, progress);
 
-	if (luma <= time - softness)
-		return b_color;
+	if (luma <= time - soft){
```

Comment:
Style:  Place a space between the closing parenthesis and the opening curly brace.

```suggestion
	if (luma <= time - soft) {
```